### ISSUE TITLE
[A11yMAS][Accessibility][Keyboard][MAS2.1.1][Eclipse][Wipro]: Help button is  not accessible by using keyboard in "Available Software" window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1350]
**Repro Steps:**

1. Open and Start Eclipse.
2. Press "Alt+H" to open Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4.  In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Then tab up to Tab  up to "Contact all update sites during install to find required software" checkbox then try to access "Help button and observe the keyboard behavior

**Note:** This issue is observed across all the windows in Install window.(Install Dialog and Review License Dialog)

**Expected Result:**
Help button should be accessible using keyboard

**Actual Result:**
Help button is not accessible using keyboard

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(OS Build 16299.192)

![unable to access the help button in available software](https://user-images.githubusercontent.com/22956130/35389457-c6281e6e-01fd-11e8-8ee8-af5029042b1a.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=534670

**Comment by jdneo** [comment]
The help page can be accessed by hitting `F1`:

![help](https://user-images.githubusercontent.com/6193897/43438959-56da38e0-94c3-11e8-9853-86c23d3bb2b7.gif)

The Eclipse version I'm using is `Oxygen.3a Release (4.7.3a)`

**Comment by WiproPSAccTest** [comment]
Verified the issue in Version Oxygen 3a Release (4.7.3a) .Issue got fixed .Please add closed tag .

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749829
